examples/amp-story/ads/app-install.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
```

Comment:
How is this example related to the fake ads? 🤔 No worries if it isn't just curious 😄

Reply:
This is the fake ad that gets loaded.

Reply:
Oh haha! Rad! Sounds good 😄